### PR TITLE
Ana/fix mixed content validator pictures over http

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Should fix the mixed content making our webapp have "Insecure" connection when connecting to Substrate networks.

I still need to try the scriptRunner `replace` fix 

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f24237fd17826a2e8663d1e--lunieio.netlify.app

File: scriptRunner/src/keybase.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ module.exports.getKeybaseImages = async () => {
             return {
               name,
               operator_address,
-              picture: result.list[0].keybase.picture_url
+              picture: result.list[0].keybase.picture_url.replace('http', 'https')
```
Comment (faboweb) [inline scriptRunner/src/keybase.js:31]: What happens with already stored images?

Comment (Bitcoinera) [reply scriptRunner/src/keybase.js:31]: I fixed them with this Postgres command: 
`update polkadot.validatorprofiles  set picture = replace(picture, 'http://', 'https://');`

Comment (faboweb) [reply scriptRunner/src/keybase.js:31]: Perfecto

**Pull request merged**